oopIntro/chair_module.py

```diff
@@ -0,0 +1,70 @@
+import time
+from ShowFunc import show
+
+# A
+
+
+class Gun:
+
+    # The constructor function
+    def __init__(self, ammunition: int, model: str):
+        if ammunition < 0 or ammunition > 6:
+            print("Error! Please enter a number between [0,6]")
+        else:
+            self.ammunition = ammunition
+        if not (model == "Sniper" or model == "Assault" or model == "Automatic"):
+            print("Error! Please enter a valid type of gun")
+        else:
+            self.model = model
+
+    def reload_magazine(self):
+        '''
+        Reload ammunition to its max capacity
+        '''
+        self.ammunition = 6
+        return self.ammunition
+
+    def get_ammu(self) -> int:
+        '''
+        Returns the current ammunition count
+        '''
+        return self.ammunition
+
+    def get_model(self) -> str:
+        '''
+        Returns the gun model
+        '''
+        return self.model
+
+    def is_empty(self):
+        '''
+        Returns True if ammunition is empty and false otherwise
+        '''
+        if self.ammunition == 0:
+            return True
+        else:
+            return False
+
+    def shoot(self):
+        '''
+        A function which cause the gun to shoot
+        '''
+        if (self.ammunition != 0):
```

Comment:
Why the parentheses?

Reply:
Changed. There are no parentheses in 'if else' statements at all